vendor/github.com/elastic/gosigar/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
```

Comment:
I tagged the release with `v0.3.0`.

If you use `govendor fetch github.com/elastic/gosigar/...@v0.3.0` it will update all gosigar and add the tag info to the vendor file.